crypto/secp256k1/secp256k1.go

```diff
@@ -9,13 +9,13 @@ import (
 	"math/big"
 
 	secp256k1 "github.com/btcsuite/btcd/btcec"
-	"golang.org/x/crypto/ripemd160" // nolint: staticcheck // necessary for Bitcoin address format
+	"golang.org/x/crypto/ripemd160" //nolint: staticcheck // necessary for Bitcoin address format
 
 	"github.com/tendermint/tendermint/crypto"
 	tmjson "github.com/tendermint/tendermint/libs/json"
 )
 
-//-------------------------------------
+// -------------------------------------
```

Comment:
```suggestion
```

Annotating a code with large set of horizontal lines is a weird choice. I'd just delete the comment because it's a bit silly and doesn't help really read this file.